devices.js

```diff
@@ -79,6 +81,55 @@ const waveshare7in2v2Vertical = {
         this.driver.init();
     },
 };
+const renderCanvas = () => {
+    const canvas = createCanvas(200, 200);
+    const ctx = canvas.getContext('2d');
+
+    // Write "Awesome!"
+    ctx.font = '30px Impact';
+    ctx.rotate(0.1);
+    ctx.fillText('Awesome!', 50, 100);
+
+    // Draw line under text
+    var text = ctx.measureText('Awesome!');
+    ctx.strokeStyle = 'rgba(0,0,0,0.5)';
+    ctx.beginPath();
+    ctx.lineTo(50, 102);
+    ctx.lineTo(50 + text.width, 102);
+    ctx.stroke();
+
+    return canvas.toBuffer();
+};
+
+const waveshare7in5bHDHorizontal = {
+    height: 528,
+    width: 880,
+    driver: waveshare7in5bHDDriver,
+    displayPNG: async function (imgContents, redImgContents) {
```

Comment:
@ascension for the other displays, this method is called in `page.js:28` where a PNG buffer browser screenshot is passed in as a single parameter. How was your workflow with these two parameters `imgContents` and `redImgContents`?